app/views/passengers/_form.html.haml

```diff
@@ -24,18 +27,29 @@
           :id, :name, @passenger.mobility_device&.id), include_blank: 'None'
 
       = form.form_group do
-        = form.check_box :active?, {checked: true}, 'active', 'archived'
+        = form.check_box :permanent, disabled: !@current_user.admin?
 
-      .verification-fields{class: doctors_note_fields_class}
-        = form.form_group do
-          = form.check_box :has_doctors_note,
-            checked: @passenger.eligibility_verification.present?
+      .verification-fields
         = form.fields_for :eligibility_verification, @verification do |d|
+          = d.select :verifying_agency_id,
+            options_for_select(VerifyingAgency.all.map { |agency| [agency.name,
+            agency.id, { 'data-needs-contact-info': agency.needs_contact_info? } ] },
+            @passenger.eligibility_verification&.verifying_agency&.id),
+            help: 'required only for temporary passengers',
+            include_blank: true
           - if @passenger.eligibility_verification.blank? && @passenger.temporary?
             - expiration_help = "Rides expire #{@passenger.rides_expire.try(:strftime, '%m/%d/%Y')}"
           = d.text_field :expiration_date,
             class: 'datepicker verification-expires',
-            help: expiration_help
+            help: expiration_help,
+            disabled: @passenger.permanent?
+          = d.form_group :contact_information,
+            label: { text: 'This verifying agency requires contact information to be entered' },
+            class: contact_information_class(@verification),
+            help: 'either phone or address is required' do
```

Comment:
Both is fine, too?

Reply:
Yes but how do I say that without being way too wordy?
I do think a reasonable human would go with "well both is probably fine" rather than reading "but not both" from what's written... whatever that's worth.

Reply:
Yeah, you're right.